guides/common/modules/ref_utilities-for-collecting-log-information.adoc

```diff
@@ -15,27 +20,34 @@ Additionally, the `foreman-debug` command exports tasks run during the last 60 d
 By default, the output tar file is located at `/tmp/__task-export-xxx.tar.xz__`.
 If the file is missing, see the file `/tmp/task-export.log` to learn why task export was unsuccessful.
 
-For more information, run `foreman-debug --help`.
+ifndef::orcharhino[]
+For more information, run `foreman-debug -h`.
+endif::[]
+ifdef::orcharhino[]
+For more information, run `orcharhino-debug -h`.
+endif::[]
 
 There is no timeout when running this command.
+ifdef::satellite[]
 |`sosreport` |The `sosreport` command is a tool that collects configuration and diagnostic information from a Red{nbsp}Hat Enterprise{nbsp}Linux system, such as the running kernel version, loaded modules, and system and service configuration files.
 The command also runs external programs (for example: `foreman-debug -g`) to collect {Project}-specific information, and stores this output in a tar file.
 
 By default, the output tar file is located at `/var/tmp/__sosreport-XXX-20171002230919.tar.xz__`.
-ifndef::orcharhino[]
 For more information, run `sosreport --help` or see https://access.redhat.com/solutions/3592[_What is a sosreport and how can I create one?_].
-endif::[]
-ifdef::orcharhino[]
-For more information, run `sosreport --help`.
-endif::[]
 
 The `sosreport` command calls the `foreman-debug -g` and times out after 500 seconds.
 If your {ProjectServer} has large log files or many {Project} tasks, support engineers may require the output of `sosreport` and `foreman-debug` when you open a support case.
+endif::[]
 |===
 
 [IMPORTANT]
 ====
+ifndef::satellite[]
+`foreman-debug` removes security information such as passwords, tokens, and keys while collecting information.
+endif::[]
+ifdef::satellite[]
```

Comment:
Please adjust these ifdefs to match as suggested above.

Reply:
Applied.